deployment/stacks/data.ts

```diff
@@ -0,0 +1,91 @@
+import { Construct } from "constructs";
+import {
+  aws_dynamodb as dynamodb,
+  aws_iam as iam,
+  aws_s3 as s3,
+  CfnOutput,
+  Aws,
+} from "aws-cdk-lib";
+import { DynamoDBTable } from "../constructs/dynamodb-table";
+
+interface CreateDataComponentsProps {
+  scope: Construct;
+  stage: string;
+  isDev: boolean;
+  loggingBucket: s3.IBucket;
+}
+
+export function createDataComponents(props: CreateDataComponentsProps) {
+  const { scope, stage, isDev, loggingBucket } = props;
+
+  const tables = [
+    new DynamoDBTable(scope, "Banner", {
+      stage,
+      isDev,
+      name: "banners",
+      partitionKey: { name: "key", type: dynamodb.AttributeType.STRING },
+    }).identifiers,
+    new DynamoDBTable(scope, "FormTemplateVersions", {
+      stage,
+      isDev,
+      name: "form-template-versions",
+      partitionKey: { name: "reportType", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "versionNumber", type: dynamodb.AttributeType.NUMBER },
+      lsi: [
+        {
+          indexName: "LastAlteredIndex",
+          sortKey: { name: "lastAltered", type: dynamodb.AttributeType.STRING },
+        },
+        {
+          indexName: "IdIndex",
+          sortKey: { name: "id", type: dynamodb.AttributeType.STRING },
+        },
+        {
+          indexName: "HashIndex",
+          sortKey: { name: "md5Hash", type: dynamodb.AttributeType.STRING },
+        },
+      ],
+    }).identifiers,
+    new DynamoDBTable(scope, "SarReports", {
+      stage,
+      isDev,
+      name: "sar-reports",
+      partitionKey: { name: "state", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "id", type: dynamodb.AttributeType.STRING },
+    }).identifiers,
+    new DynamoDBTable(scope, "WpReports", {
+      stage,
+      isDev,
+      name: "wp-reports",
+      partitionKey: { name: "state", type: dynamodb.AttributeType.STRING },
+      sortKey: { name: "id", type: dynamodb.AttributeType.STRING },
+    }).identifiers,
+  ];
+
+  const sarFormBucket = new s3.Bucket(scope, "SarFormBucket", {
+    bucketName: `database-${stage}-sar`,
+    encryption: s3.BucketEncryption.S3_MANAGED,
+    versioned: true,
+    blockPublicAccess: s3.BlockPublicAccess.BLOCK_ALL,
+    serverAccessLogsBucket: loggingBucket,
+    serverAccessLogsPrefix: `AWSLogs/${Aws.ACCOUNT_ID}/s3/`,
```

Comment:
I'd save these (and same for other spots) for a future ticket where we add this to all repos. Just trying to keep these in sync between repos.

Reply:
Are you saying to remove for now?  I added these to match MFP as-is.

Reply:
I think it's fine to leave in if we're maintaining what was before. I've made a note already to figure out whether we need this or not and standardize later so we are consistent across apps.